ibis/backends/bigquery/__init__.py

```diff
@@ -125,6 +125,27 @@ def _remove_null_ordering_from_unsupported_window(
     return node
 
 
+def _force_quote_table(table: sge.Table) -> sge.Table:
+    """Force quote all the parts of a bigquery path.
+
+    The BigQuery identifier quoting semantics are bonkers
+    https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#identifiers
+
+    my-table is OK, but not mydataset.my-table
```

Comment:
What would the `my-table` unqualified with a dataset even mean?

Reply:
I don't know -- why would Google include that example in their documentation?

Reply:
LOL